src/Expr.js

```diff
@@ -263,6 +249,70 @@ var exprToString = function(expr, caller) {
     )
   }
 
+  if ('collections' in expr) {
+    if (expr['collections'] === null) {
+      return 'Collections()'
+    }
+  }
+
+  if ('databases' in expr) {
+    if (expr['databases'] === null) {
+      return 'Databases()'
+    }
+  }
+
+  if ('select' in expr) {
```

Comment:
doesn't need special case too, also, if it's needed for some reason, the function is missing the `default` argument

Reply:
@marrony Same here -- this is failing:
![image](https://user-images.githubusercontent.com/4997781/94149059-6b066080-fe45-11ea-992c-37f19f53b708.png)

Reply:
yeah, but your test `pretty print despite argument order` is not valid anymore right, given core will always the objects in the same order, also, we reverted most of the "order agnostic" code, those one that still remain is to fix or add a custom rendering, like `map`, `foreach` and `filter` for example